### PR TITLE
Feed tags update

### PR DESCRIPTION
**Pull request by orhovy**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://github.com/demisto/etc/issues/22895

## Description
updated additional info and added feedtags params to feeds which have the implementation of it in the code.

## Must have
- [ ] Tests
- [ ] Documentation 

## Demisto Partner?
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**

### CODE REVIEW

File: Packs/AutoFocus/ReleaseNotes/1_1_1.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+##### AutoFocus Feed
+-
```
Comment (bakatzir) [inline Packs/AutoFocus/ReleaseNotes/1_1_1.md:4]:
```suggestion
Improved parameter description.
```

File: Packs/AutoFocus/ReleaseNotes/1_1_1.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+##### AutoFocus Daily Feed
+-
```
Comment (bakatzir) [inline Packs/AutoFocus/ReleaseNotes/1_1_1.md:6]:
```suggestion
Improved parameter description.
```

File: Packs/FeedJSON/ReleaseNotes/1_0_2.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### JSON Feed
+-
```
Comment (bakatzir) [inline Packs/FeedJSON/ReleaseNotes/1_0_2.md:4]:
```suggestion
Improved parameter description.
```

File: Packs/FeedTAXII/ReleaseNotes/1_0_3.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### TAXII 2 Feed
+- Added the *Tags* parameter.
```
Comment (bakatzir) [inline Packs/FeedTAXII/ReleaseNotes/1_0_3.md:4]: We already had it, just a change of additionalinfo
```suggestion
Improved parameter description.
```

File: Packs/FeedTorExitAddresses/ReleaseNotes/1_0_1.md [hunk 5]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Tor Exit Addresses Feed
+-
```
Comment (bakatzir) [inline Packs/FeedTorExitAddresses/ReleaseNotes/1_0_1.md:4]:
```suggestion
Improved parameter description.
```

File: Packs/FeedUnit42/ReleaseNotes/1_0_1.md [hunk 6]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Unit42 Feed
+-
```
Comment (bakatzir) [inline Packs/FeedUnit42/ReleaseNotes/1_0_1.md:4]:
```suggestion
Improved parameter description.
```

**Comment by orhovy** [comment]
There is no change in the code for those because they all inherit from jsonapi or csvapi which already implements tags in params.

File: Packs/AutoFocus/ReleaseNotes/1_1_1.md (hunk 1 above)
Comment (amshamah419) [reply Packs/AutoFocus/ReleaseNotes/1_1_1.md:4]: @orhovy You can apply Bar's suggestions to all of the release notes.

**Comment by orhovy** [comment]
TAXII 2 Feed Test
AutoFocus V2 test
Unit42 Feed - Test
Tests playbook fail on 5.5 and 6 however the changes of this pr are not supposed to be the reason.
Please Force merge

**Pull request merged**